vm/src/host_queries.rs

```diff
@@ -8,6 +8,7 @@
 
 use alloc::vec::Vec;
 
+use crate::cache;
```

Comment:
nit nit: this needs to be on line 35, below the sha3 import, separated by a new line